packages/website/blog/2024-03-01-parser-options-project-service.md

```diff
@@ -0,0 +1,298 @@
+We measured several common project scales:
+
+- 👶 Linting a "tiny" project (~dozen files)
+- 🧒 Linting a "small" project (~100 files)
+- 🧑 Linting a "medium" project (~1,000 files)
+- 🧓 Linting a "large" project (~5,000 files, with project references)
+
+We also measured across different linting scenarios:
+
+- "Full cold start" of linting the entire project
+- "Partial cold start" of seeing editor suggestions after opening the project in VS Code
+- "Warm start" of seeing editor suggestions after changing a file in VS Code
+
+See [typescript-eslint/project-apis-performance-comparison](https://github.com/typescript-eslint/project-apis-performance-comparison) for details on the performance measurements.
+
+### Full Cold Start
+
+These measurements were taken by running `npm run lint` on the respective projects.
+
+| API                | 👶 Tiny | 🧒 Small | 🧑 Medium | 🧓 Large |
+| ------------------ | ------- | -------- | --------- | -------- |
+| Legacy Program     |         |          |           |          |
+| 🆕 Project Service |         |          |           |          |
+| 𝚫 Delta            |         |          |           |          |
+
+### Partial Cold Start
+
+These measurements were taken by recording restarting VS Code on the respective projects with a single file open.
+
+| API                | 👶 Tiny | 🧒 Small | 🧑 Medium | 🧓 Large |
+| ------------------ | ------- | -------- | --------- | -------- |
+| Legacy Program     |         |          |           |          |
+| 🆕 Project Service |         |          |           |          |
+| 𝚫 Delta            |         |          |           |          |
+
+### Warm Start
+
+These measurements were taken by recording VS Code on the respective projects with a single file open and modifying that file.
+
+| API                | 👶 Tiny | 🧒 Small | 🧑 Medium | 🧓 Large |
+| ------------------ | ------- | -------- | --------- | -------- |
+| Legacy Program     |         |          |           |          |
+| 🆕 Project Service |         |          |           |          |
+| 𝚫 Delta            |         |          |           |          |
+
+## Next Steps
+
+### Giving Feedback
+
+We'd love to hear from you on how this option works for you.
+Does it live up to what we've promised, and/or does it have bugs we haven't fixed yet?
+Please do post in the [Community Feedback: Project Service APIs](https://github.com/typescript-eslint/typescript-eslint/discussions/8030) GitHub Discussions post on how it goes for you.
+
+For support in setting up the new APIs, feel free to ask on [the typescript-eslint Discord](https://discord.gg/FSxKq8Tdyg)'s `#project-service` channel.
+We'd be happy to help you try out `parserOptions.projectService`.
+
+### Long Term Vision
+
+Our hope is that the Project Service API becomes the standard way to work with typed linting over the next few major versions.
+Our intent is to roll it out according to the following rough timeline:
```

Comment:
In theory we could rename to `projectService` in v6. We did say we'd keep it `EXPERIMENTAL_` for all of v6, but we can always add an alias. Being ahead of schedule isn't a problem as long as we don't cause extra work we promised not to. 😄

My main gripe with renaming in v6 rather than v7 is that it convolutes the messaging around the upgrade. For simplicity's sake I'm leaning towards taking our time. But I'm not hard-set on this.